ts/screens/onboarding/EmailInsertScreen.tsx

```diff
@@ -81,11 +82,22 @@ class EmailInsertScreen extends React.PureComponent<Props, State> {
 
   /**
    * Footer
+   *
+   * TODO1: add navigation to the dedicated modal
+   *          https://www.pivotaltracker.com/story/show/168247501
+   * TODO:2 save the inserted new email
+   *          https://www.pivotaltracker.com/n/projects/2048617/stories/169264055
    */
   private renderFooterButtons() {
     const continueButtonProps = {
       disabled: this.isValidEmail() !== true,
-      onPress: this.props.dispatchEmailInsert,
+      onPress: this.isEditing
+        ? () => {
+            // TODO1
+            // TODO2
+            Alert.alert("Not implemented");
```

Comment:
Despite it is under development use language localization

Reply:
updated within updated within https://github.com/teamdigitale/io-app/pull/1363/commits/2d1442691356a14fc1b5864a0a30220da184f8cb